integration-tests/mockApis/caseNotes.js

```diff
@@ -14,11 +14,29 @@ const caseNoteTypes = [
       },
     ],
   },
+  {
+    code: 'OMIC',
+    description: 'OMiC',
+    activeFlag: 'Y',
+    source: 'OCNS',
+    subCodes: [
+      {
+        code: 'OPEN_COMM',
+        description: 'Open Case Note',
+        activeFlag: 'Y',
+      },
+      {
+        code: 'COMM',
+        description: 'OMiC Communication',
+        activeFlag: 'Y',
+      },
+    ],
+  },
 ]
 
 module.exports = {
-  stubHealth: (status = 200) => {
```

Comment:
I just reformatted the file with prettier - looks like it hadn't been done before :shrug: